### ISSUE TITLE
Error while installing

### ISSUE THREAD

**Issue by bumothekid** [wakatime/xcode-wakatime#62]
I have an issue that when I try to install the plugin for Xcode with the command
```
curl -fsSL https://raw.githubusercontent.com/wakatime/xcode-wakatime/master/install.sh | sh
```
it throws the error
```
tar: Error exit delayed from previous errors.
```
this error occurs while it downloads "Alcatraz" 

Here's also an screenshot:

<img width="749" alt="Screenshot 2022-05-07 at 1 20 02 PM" src="https://user-images.githubusercontent.com/40246197/167252044-31d48418-090f-4834-b730-2fd885d6573e.png">

**Comment by alanhamlett** [comment]
Fixed with 2ba57b878b1cb01c51f0d34280711f94a5a8bb9c.

**Comment by bumothekid** [comment]
Thanks
